Assets/Scripts/Interactables/Interactable.cs

```diff
@@ -1,36 +1,66 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+[RequireComponent(typeof(AudioSource))]
 public class Interactable : MonoBehaviour
 {
-  public bool allowMultipleInteractions = false;
-  IrritationBar irritationBar;
-  InteractableUI interactableUI;
+  [SerializeField] private bool allowMultipleInteractions = false;
+  [SerializeField] private List<IrritationReaction> irritationReactionList = new List<IrritationReaction>();
+  [SerializeField] private InteractableAudio audioClips;
+  [SerializeField] private InteractablePlayerAnimation playerAnimations;
 
   [System.Serializable]
   public struct IrritationReaction
   {
     public NonPlayerController nonPlayerController;
     public float irritationModifier;
   }
-  public List<IrritationReaction> irritationReactionList = new List<IrritationReaction>();
-  NonPlayerController reactingNPC;
 
-  [SerializeField] private PlayerAnimation mouthPlayerAnimation;
-  [SerializeField] private PlayerAnimation pawPlayerAnimation;
-  [SerializeField] private PlayerAnimation bodyPlayerAnimation;
+  private IrritationBar irritationBar;
+  private InteractableUI interactableUI;
+  private NonPlayerController reactingNPC;
+  private AudioSource audioSource;
 
-  void Start()
+  protected virtual void Start()
   {
     interactableUI = GetComponent<InteractableUI>();
+    audioSource = GetComponent<AudioSource>();
```

Comment:
I moved public methods to the top of the file, so it's a little easier to see the public interface at a glance